lib/types/webp.js

```diff
@@ -10,10 +10,21 @@ function isWebP (buffer) {
 }
 
 function calculate (buffer) {
-
   var chunkHeader = buffer.toString('ascii', 12, 16);
   buffer = buffer.slice(20, 30);
 
+  // Extended webp stream signature
+  if (chunkHeader === 'VP8X') {
+    var extendedHeader = buffer[0];
+    var validStart = (extendedHeader & 0b11000000) === 0;
```

Comment:
The reason of [this CI failure](https://travis-ci.org/image-size/image-size/jobs/182573803#L225) is [escomplex-js](https://github.com/jared-stilwell/escomplex-js) does not support binary literals.

I think now is the time to remove escomplex-js from this project. escomplex-js is a Node.js wrapper of no longer maintained [escomplex](https://github.com/escomplex/escomplex).